bundles/org.openhab.binding.homeconnect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,664 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="homeconnect"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Home Connect API Bridge -->
+	<bridge-type id="api_bridge">
+		<label>Home Connect API</label>
+		<description>This bridge represents the gateway to the Home Connect API.</description>
+		<config-description>
+			<parameter name="clientId" type="text" required="true">
+				<label>Client id</label>
+				<description>Application client id</description>
+			</parameter>
+			<parameter name="clientSecret" type="text" required="true">
+				<label>Client secret</label>
+				<description>Application client secret</description>
+			</parameter>
+			<parameter name="simulator" type="boolean" required="true">
+				<label>Use simulator environment</label>
+				<description>Use simulated environment at https://developer.home-connect.com/simulator/</description>
+				<default>false</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<!-- Dishwasher -->
+	<thing-type id="dishwasher">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="api_bridge"/>
+		</supported-bridge-type-refs>
+		<label>Dishwasher</label>
+		<description>Home Connect connected dishwasher (e.g. Bosch or Siemens).</description>
+		<channels>
+			<channel id="power_state" typeId="power_state"/>
+			<channel id="door_state" typeId="door_state"/>
+			<channel id="operation_state" typeId="operation_state"/>
+			<channel id="remote_start_allowance_state" typeId="remote_start_allowance_state"/>
+			<channel id="remote_control_active_state" typeId="remote_control_active_state"/>
+			<channel id="selected_program_state" typeId="selected_program_state"/>
+			<channel id="basic_actions_state" typeId="basic_actions_state"/>
+			<channel id="active_program_state" typeId="active_program_state"/>
+			<channel id="remaining_program_time_state" typeId="remaining_program_time_state"/>
+			<channel id="program_progress_state" typeId="program_progress_state"/>
+			<channel id="ambient_light_state" typeId="ambient_light_state"/>
+			<channel id="ambient_light_brightness_state" typeId="ambient_light_brightness_state"/>
+			<channel id="ambient_light_color_state" typeId="ambient_light_color_state"/>
+			<channel id="ambient_light_custom_color_state" typeId="ambient_light_custom_color_state"/>
+		</channels>
+		<config-description>
```

Comment:
Please add `<representation-property>haId</representation-property>` just before config-description.
Same in all things types.

Reply:
Will adjust discovery service as well